WordPressKitTests/TransactionsServiceTests.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+import WordPressShared
+import CocoaLumberjack
+@testable import WordPressKit
+
+class TransactionsServiceTests: RemoteTestCase, RESTTestable {
```

Comment:
👍 for adding new tests!!

Looking at other remote tests, it looks like they all have `remote` in the class name and file name. Can we rename this to something like `TransactionsServiceRemoteTests`?

Reply:
Sure